webrtc-stats.html

```diff
@@ -1521,20 +1521,36 @@
           <li>Clarified that bytesSent and bytesReceived do no include headers or paddings.
           </li>
         </ol>
-        <section id="since-3-feb-2015*">
-          <h3>
-            Changes since 03 February 2015
-          </h3>
-          <ol>
-            <li>Added RTCRTPStreamStats.mediaType (jan-ivar, PR #10)
-            </li>
-          </ol>
-        </section>
-        <section id="since-23-oct-2015*">
-          <h3>
-            Changes since 23 October 2015
-          </h3>
-        </section>
+      </section>
+      <section id="since-3-feb-2015*">
+        <h3>
+          Changes since 03 February 2015
+        </h3>
+        <ol>
+          <li>Added RTCRTPStreamStats.mediaType (jan-ivar, PR #10)
```

Comment:
based on the below structure, shouldnt it be: 
[#10] Added RTCRTPStreamStats.mediaType (jan-ivar)